### DISCUSSION TITLE
About sys commands like Sys.exit and Sys.environment.

### DISCUSSION THREAD

**Discussion by ghost** [HaxeFlixel/flixel#2623] [Q&A] [answered]
I'm trying to find some tutorials about sys commands, but can't find any. Is there a list of sys commands and functions anywhere?

**Comment by Geokureli** [comment, marked as answer]
You can find all the Sys methods, here: https://api.haxe.org/Sys.html

BTW this is a haxe question, not a HaxeFlixel question, for haxes standard packages you can always check the haxe docs, or google "haxe sys" or w/e

**Reply by ghost** [reply]
Thanks!
